### DISCUSSION TITLE
small test / dataset item

### DISCUSSION THREAD

**Discussion by culurciello** [NVlabs/instant-ngp#801] [General]
Hi, I wonder if anyone has a smaller example for NERF than https://github.com/nickponline/dd-nerf-dataset maybe just 10-20 images or so or a boxy object / building, just to test on a smaller item.
If you do / know let me know

**Comment by 3a1b2c3** [comment]
You saw the fox included in data folder? Otherwise just delete data from the .json

**Comment by culurciello** [comment]
thanks a lot

**Reply by nickponline** [reply]
@culurciello The DroneDeploy dataset has a lot of images for each scene but the default json script only uses the 50 closest to the center of the scene.
